Client/Writes/PointData.cs

```diff
@@ -53,9 +66,23 @@ public static PointData Measurement(string measurementName)
         /// <returns>this</returns>
         public PointData Tag(string name, string value)
         {
-            _tags[name] = value;
+            if (string.IsNullOrEmpty(value))
```

Comment:
We should enable add tag with `null_or_empty` value. The source of data for the `PointData` could be third party services and we don't know what they are want.

This test should pass:

```csharp
[Test]
public void TagEmptyTagValue()
{
    var point = PointData.Measurement("h2o")
        .Tag("location", "europe")
        .Tag("log", "to_delete")
        .Tag("log", "")
        .Field("level", 2);

    Assert.AreEqual("h2o,location=europe level=2i", point.ToLineProtocol());
}
```

Reply:
OK checking it